libraries/rush-lib/src/api/Subspace.ts

```diff
@@ -56,6 +61,23 @@ export class Subspace {
     return this._projects;
   }
 
+  public getPnpmOptions(): PnpmOptionsConfiguration | undefined {
+    if (!this._pnpmOptions) {
+      if (FileSystem.exists(`${this.getSubspaceConfigFolder()}/${RushConstants.pnpmConfigFilename}`)) {
```

Comment:
Instead of doing an existence check, consider changing the load function to return undefined if the file doesn't exist. It can do that by trying to load the file and checking if an exception thrown is a not-exist error.

Reply:
Changed